core/validation/messages.py

```diff
@@ -8,6 +8,7 @@
 )
 BLANK_ZERO = BLANK + " Enter a value, even if it’s zero."
 BLANK_PSI = BLANK + " Enter why the private sector investment gap is greater than zero."
+BLANK_SIGN_OFF = "In tab '8 - Review & Sign-Off', cell {cell} is blank but is required."
```

Comment:
Could you run this by Alice quickly?

Reply:
Ah, she is off. Lets get this in for now but could you message and revisit tomorrow if it needs to be altered?

Reply:
Will do